myapp/config/routes.rb

```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  scope(path_name: {index: ""}) do
+    resources :tasks, path: "/"
+  end
```

Comment:
Simpleにこれでいいかもです
```suggestion
  root 'tasks#index'
  resources :tasks
```
https://railsguides.jp/routing.html#root%E3%82%92%E4%BD%BF%E3%81%86

Reply:
As you can see the pic below, there will be a `tasks/` prefix in the url, and in this training's case I don't think this prefix is necessary so I removed it intentionally.

![image](https://github.com/Fablic/training/assets/167514427/c62aeef0-1347-47b2-95cc-d31a018bb6c5)

Reply:
I understand your intention! Let's see if we really don't need `/tasks` in path names 😄